### DISCUSSION TITLE
【冒泡】各位都是B站来的嘛？

### DISCUSSION THREAD

**Discussion by Rrrr-iknn** [share121/inter-knot#45] [其他]
![微信图片_202402061258311](https://github.com/user-attachments/assets/10ba1270-6d92-4452-afb8-00331168b4ba)
刚刚看了绳网跨域助手下载量才72，而B站播放量已经有3万了

**Comment by ghost** [comment]
我确实是b站来的，要不说这做的还挺好。

**Comment by INAhi-inako** [comment]
嗯呢嗯呢（是的）

**Comment by RS64671012** [comment]
嗯呢嗯呢

**Reply by zhuxuepengcha** [reply]
是这样的，还在摸索怎么使用，以及作者真棒

**Comment by MYfavoriterose** [comment]
嗯呐嗯呐

**Comment by QAQ-114** [comment]
嗯呐嗯呐

**Comment by myt521** [comment]
嗯呢嗯呢

**Comment by byHydrogen** [comment]
嗯呢嗯呢

**Comment by Luli-Chen** [comment]
嗯呢嗯呢

**Comment by lingkunya** [comment]
这年头，帮布也能上绳网吗？

**Comment by gaojiaobei** [comment]
嗯呢嗯呢

**Comment by minyue-yu** [comment]
嗯呢嗯呢

**Comment by kaijinti** [comment]
进来一看被邦布占领了

**Comment by niegong** [comment]
嗯呢嗯呢

**Comment by 765hi** [comment]
嗯呢嗯呢

**Comment by Drs1145** [comment]
是的捏

**Comment by kimi12123** [comment]
嗯呢嗯呢

**Comment by nanamiitsugi** [comment]
嗯呢嗯呢

**Comment by uyfh352** [comment]
嗯呢嗯呢

**Comment by g98054** [comment]
嗯呢嗯呢

**Comment by Sushizhu** [comment]
嗯呢嗯呢

**Comment by Qiling-XwX** [comment]
嗯呢嗯呢

**Comment by axuanran** [comment]
嗯呢嗯呢

**Comment by shuyishaoxiancao** [comment]
当然

**Comment by Elysia-Ego-093** [comment]
嗯呢嗯呢！(当然！)

**Comment by szlx2333** [comment]
嗯呢！

**Comment by WASDDestroy** [comment]
**嗯呢嗯呢**

**Comment by Ryan83602** [comment]
嗯呢！

**Comment by Blazing-dust** [comment]
嗯呢嗯呢！

**Comment by Dream-Alan** [comment]
嗯呢嗯呢

**Comment by songxu0012** [comment]
上面那些不像是b站来的，应该是哪家邦布工厂跑出来的

**Comment by lumao1106** [comment]
嗯呢嗯呢

**Comment by su2569** [comment]
嗯呢嗯呢

**Comment by trv3wood** [comment]
嗯呢嗯呢

**Comment by asderfgh** [comment]
嗯呢嗯呢

**Comment by celloonly** [comment]
嗯呐哒 嗯呐哒

**Comment by Administrator1145** [comment]
嗯呢嗯呢

**Comment by baize000** [comment]
是的

**Comment by Furina-D** [comment]
嗯呢嗯呢

**Comment by BITE-7274** [comment]
嗯呢嗯呢

**Comment by Friskdrink** [comment]
嗯呐嗯呐

**Comment by cpu644** [comment]
嗯呢嗯呢

**Comment by wdnmd7758** [comment]
包的

**Comment by Shiershi** [comment]
嗯呢嗯呢

**Comment by KoaSayi** [comment]
嗯呢嗯呢！

**Comment by GanlinSama** [comment]
嗯呢嗯呢（包的兄弟）

**Comment by ei-h9** [comment]
![172224010806894cb56e5c9d0d67e9dcce1374861cadc969aa5d5b5a029faebe1199745eda678 0](https://github.com/user-attachments/assets/c83a0a6d-0723-4fca-ba0b-0976b04b3510)
是的

**Reply by Hongxuli2006** [reply]
翻过一堆文字看到你这个请允许我突然笑了

**Comment by YUAN-DAT** [comment]
嗯呐嗯呐

**Comment by fairy-B** [comment]
嗯呐嗯呐！还是挺完善的，但是为什么人那么少
![image](https://github.com/user-attachments/assets/885c39fa-7765-4d3f-9b38-a87214c24a59)
还有这些977是什么？？？

**Reply by ei-h9** [reply]
这个是浏览人数，但目前是随机的，还没做好

**Reply by Hongxuli2006** [reply]
等慢慢优化吧，那个数字挺奇怪的

**Comment by CW10086114404** [comment]
嗯呢嗯呢

**Comment by Mayf1ower** [comment]
嗯呢嗯呢

**Comment by yoimiya-0621** [comment]
是的

**Comment by YukariZZZ** [comment]
嗯呐嗯呐

**Comment by EI-HEI-123** [comment]
嗯呢嗯呢

**Comment by Elysia-lover** [comment]
嗯呢嗯呢~

**Comment by LHk416** [comment]
嗯呢嗯呢~

**Comment by 147258huan** [comment]
对

**Comment by Yeshi299** [comment]
嗯呢嗯呢~~

**Comment by biejian-yuanshen** [comment]
![356819942-c83a0a6d-0723-4fca-ba0b-0976b04b3510](https://github.com/user-attachments/assets/539ebbd9-244b-43cb-a720-3dedb2322543)
这么还闹邦布了

**Comment by 10kaihua** [comment]
嗯呢嗯呢

**Comment by Cartion101** [comment]
嗯呢嗯呢

**Comment by 123456wanh** [comment]
嗯呢嗯呢

**Comment by convant** [comment]
嗯呢嗯呢

**Comment by Dream-Dyer** [comment]
嗯呢嗯呢

**Comment by fanvawsl** [comment]
嗯呢嗯呢

**Comment by lin7758258** [comment]
嗯呢嗯呢

**Comment by dtryftryu** [comment]
嗯呢嗯呢

**Comment by wuwu0621** [comment]
嗯呢嗯呢

**Comment by CocoonedSoul** [comment]
嗯呢~~嗯呢~~

**Comment by Kujojolen** [comment]
嗯呢嗯呢

**Comment by ywcSystem** [comment]
是的捏

**Comment by yaniorkoop** [comment]
嗯呢嗯呢~

**Comment by martiansky** [comment]
嗯呢嗯呢

**Comment by FFARD** [comment]
嗯呢嗯呢~

**Comment by ximulili** [comment]
嗯呢哒嗯呢哒

**Comment by canifish** [comment]
这年头邦布都能随便访问绳网了？万一邦布乱接委托岂不害惨那些进空洞的了

**Comment by natihawi** [comment]
令人深思的观看人数嗯呢....

**Comment by FurinaShark** [comment]
嗯呢嗯呢，还特意注册了一个绳网账号呢

**Comment by mobious9527** [comment]
嗯呢嗯呢！(这里好多邦布啊！)

**Comment by Sui-Qing** [comment]
嗯呢嗯呢

**Comment by BITE-7274** [comment]
![356713659-8e0dde19-d223-4675-94ab-bcfe8198092e](https://github.com/user-attachments/assets/94b00a81-87bb-4e8c-992a-577d66fbbd4e)
嗯呢嗯呢？！？嗯呢嗯，嗯呢嗯，嗯呢嗯呢！（这么多邦布啊?!?兄弟们，没钱用了，出来抓黄金邦布！）

**Comment by piaoxueol** [comment]
嗯呢嗯呢

**Comment by BearOctopus** [comment]
嗯呢嗯呢

**Comment by gdio185** [comment]
谁家邦布跑出来了

**Comment by Atsets** [comment]
嗯呢嗯呢

**Comment by shxi123** [comment]
嗯呢嗯呢

**Comment by JingJing431** [comment]
好多人呀

**Comment by yangyuting0516** [comment]
嗯呢嗯呢

**Comment by huxinchi** [comment]
嗯呢嗯呢

**Comment by breeze0317** [comment]
嗯呢嗯呢

**Comment by Fuklos** [comment]
嗯呢嗯呢

**Comment by sl44444** [comment]
嗯呐嗯呐

**Comment by Aoligei114514** [comment]
嗯呢嗯呢

**Comment by pinhexianran** [comment]
嗯呢嗯呢

**Comment by waste666** [comment]
嗯呢？嗯呢！

**Comment by hanyaser** [comment]
嗯呢呢

**Comment by bbdqz** [comment]
是的捏
![160984746](https://github.com/user-attachments/assets/e6aa48d0-a87f-423c-84f9-b9101298a3cf)
插个芙芙(我芙真好看

**Comment by H2-CO-3** [comment]
嗯呐嗯呐

**Comment by yileinafs** [comment]
还是时时的

**Comment by XeiTaiYao** [comment]
嗯呢嗯呢

**Comment by Anuaing** [comment]
是的

**Comment by jzx-176** [comment]
嗯哪嗯哪

**Comment by 12344856** [comment]
嗯呢嗯呢

**Comment by YING-ADX** [comment]
嗯呢嗯呢

**Comment by ghhccghk** [comment]
嗯呢嗯呢

**Comment by xmtx233** [comment]
嗯呢嗯呢

**Comment by richi-shek** [comment]
嗯呢嗯呢

**Comment by arez-17** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/f2f1488ffbfff3fa96d4db6047e5f409_6776930953450600054.png" alt="米仔_不了" width="75" >

**Comment by lyx1662** [comment]
嗯呢嗯呢

**Comment by wdf-86** [comment]
嗯呢嗯呢

**Comment by haloged** [comment]
嗯呢嗯呢~~~

**Comment by aimen114514** [comment]
嗯呢嗯呢

**Comment by icenow188** [comment]
是的喵

**Comment by Sheng-gy** [comment]
嗯呢嗯呢<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by FBTB-w** [comment]
嗯呢嗯呢！

**Comment by whatcanisaylaoda** [comment]
嗯呢嗯呢！

**Comment by xing-nan-4019** [comment]
嗯呢嗯呢！

**Comment by KUYUE404** [comment]
嗯呢嗯呢！<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Regionnor** [comment]
嗯呢嗯呢！（是的！）

**Comment by mingyu818** [comment]
嗯呢嗯呢

**Comment by wssjl** [comment]
嗯呢嗯呢！

**Comment by bengbu200** [comment]
嗯呢！<img src="https://bbs-static.miyoushe.com/static/2023/04/28/6475fffe75e1042eb1c68ac615adfdaf_2494846049179701109.png" alt="米仔_得意" width="75" >

**Comment by airi319** [comment]
嗯呢嗯呢！

**Comment by jianjue1** [comment]
嗯呢嗯呢！

**Comment by Usodaru48763** [comment]
嗯呢嗯呢

**Comment by 00zzxx** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >

**Comment by IHushuan** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/6475fffe75e1042eb1c68ac615adfdaf_2494846049179701109.png" alt="米仔_得意" width="75" >

**Comment by Pheathon** [comment]
嗯呐~嗯呐呐哒！（对啊）

**Comment by erwwwwwwwwyyyyyyy** [comment]
各位都是爱绳网的绳匠

**Comment by KFGAS** [comment]
嗯呢呢嗯呢

**Comment by UEG-LmR** [comment]
嗯呢嗯呢

**Comment by Natsumowari** [comment]
嗯呢嗯呢

**Comment by YuFeiQAQ** [comment]
嗯呢嗯呢

**Comment by XeBia2408** [comment]
嗯呢哒嗯呢哒<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by Andrea0126** [comment]
嗯呢嗯呢

**Comment by shengwang999** [comment]
嗯呐嗯呐<img src="https://bbs-static.miyoushe.com/static/2023/12/28/d6bfa59a83721d7e7dd27363f5bfc88b_732370910818099450.png" alt="芙宁娜-好戏！" width="75" >

**Comment by zxcvbnmqawsed123** [comment]
包的

**Comment by Thina-chng** [comment]
是的

**Comment by FishRabbit090** [comment]
<h1>嗯</h1>

**Comment by homteam** [comment]
我原本只是逛一逛绳网可是突然跳转到了这个网站，可恶，居然还有人敢做山寨版绳网<img src="https://img-static.mihoyo.com/communityweb/upload/fcb0d849f766bb544445a7ac0d5b9677.png" alt="妮可_生气" width="75" >（bushi）

**Comment by DhoomCharlie** [comment]
嗯呢嗯呢

**Comment by LilangAlan** [comment]
嗯呢嗯呢

**Comment by Lifeearning** [comment]
嗯呢嗯呢

**Comment by Abandon12393** [comment]
下载量已经有6k+了哦

**Comment by qqqqqq77777** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >

**Comment by ling086** [comment]
嗯呢嗯呢

**Comment by Fukacis** [comment]
嗯呢嗯呢

**Comment by Du380** [comment]
嗯呢嗯呢<img src="https://bbs-static.miyoushe.com/static/2023/04/28/0c624a8915dd9b24093cf630bded5748_7288657913684710277.png" alt="米仔_拜年" width="75" >

**Comment by mochengyanliu920** [comment]
嗯呢嗯呢

**Comment by 1492830678** [comment]
被某b气过来的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by YDwang1459** [comment]
包的

**Comment by Erin-yan22** [comment]
嗯呢嗯呢

**Comment by dch-cm** [comment]
❤ 加 我 看 最 新 谛 听 视 频 ❤

**Comment by mochengyanliu920** [comment]
对<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Qianxiao114** [comment]
是的

**Comment by qianchengmuxue-1997** [comment]
嗯呢嗯呢

**Comment by ailixiyachuyin** [comment]
嗯呢嗯呢

**Comment by Zou-pig** [comment]
嗯呢嗯呢呢！（是的，这个太厉害了）

**Comment by zxdgle** [comment]
嗯呢嗯呢！

**Comment by huameidudu** [comment]
嗯呢嗯呢！嗯呢哒！

**Comment by Robin-txdy** [comment]
嗯呢嗯呢！<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >

**Comment by AFUNGH** [comment]
嗯呢嗯呢
![-259e5c1b5ce30d95](https://github.com/user-attachments/assets/3e718562-2e51-40ea-8fea-6ccee0df01a1)

**Comment by FlyMan2004** [comment]
嗯呢嗯呢！

**Comment by dw1-crypto** [comment]
俺也一样🤓

**Comment by JoyChenQ** [comment]
嗯呢嗯呢

**Comment by JiuXiat** [comment]
977
